### PR TITLE
Remove flatpak-builder

### PR DESCRIPTION
**Pull request by wjt**
We take flatpak-builder directly from Debian, but maintain our own
Flatpak. Due to a security issue, flatpak-builder now has a versioned
dependency on a newer version of Flatpak than we have in the distro.

Remove flatpak-builder. We can readd it once we have updated Flatpak. In
the meantime the org.flatpak.Builder package from Flathub can be used
instead.

https://phabricator.endlessm.com/T32961

### CODE REVIEW

**Review by GeorgesStavracas — APPROVED** [review]
I don't know whether this change has deeper implications in our build stack or not, but I think this is a good idea. I'd like to have another opinion here before merging.

**Review by jprvita — APPROVED** [review]

**Comment by jprvita** [comment]
> I don't know whether this change has deeper implications in our build stack or not, but I think this is a good idea. I'd like to have another opinion here before merging.

From https://phabricator.endlessm.com/T22335#922710 I don't think it has any implication on our build infrastructure. I'm merging this on the basis that this is the easiest path to get master builds going again, and we can discuss separately whether we should re-add it an how to do it.

**Pull request merged**
